src/debug.ts

```diff
@@ -157,6 +214,32 @@ export async function debugTests(
   await deferredPromise.promise
 }
 
+function getTestProjectsInFolder(path: string, api: VitestFolderAPI, tree: TestTree) {
+  const folder = tree.getOrCreateFolderTestItem(api, path)
+  const items = tree.getFolderFiles(folder)
+  return items.map(item => (getTestData(item) as TestFile).project)
+}
+
+function getProjectsFromTests(item: vscode.TestItem | undefined, api: VitestFolderAPI, tree: TestTree): string[] {
+  const items = getTestProjectsInFolder(item?.uri?.fsPath ?? '', api, tree)
```

Comment:
It's quite hard to read a lot of `?.`, `??`, `| undefined` everywhere. Can this be simplified to only accept the value if it's there?